### DISCUSSION TITLE
filebeat module parsing [ASA firewall syslog]

### DISCUSSION THREAD

**Discussion by 73F0SG1R** [Security-Onion-Solutions/securityonion#6309] [Unsupported Versions]
ver: 2.3.80

successfully receiving asa syslog to port 514, but it isn't parsing

following "https://docs.securityonion.net/en/latest/filebeat.html#modules"

added this:

filebeat:
  third_party_filebeat:
    modules:
      cisco:
        asa:
          enabled: true
          var.input: "file"
          var.syslog_host: 0.0.0.0
          var.syslog_port: 514

to:

/opt/so/saltstack/local/pillar/minions/securityonion_standalone.sls

following instructions, did "so-filebeat-restart"

failed with:

  ID: so-filebeat
    Function: docker_container.running
      Result: False
     Comment: Created container 'so-filebeat'. Failed to start container 'so-filebeat': 'Error 500: driver failed programming external connectivity on endpoint so-filebeat (e6d8c3441f4a4622f2fe9e37d6407374084d8ddc62ddbc66589d63a8b6d49912): Bind for 0.0.0.0:514 failed: port is already allocated'.

need:
clear step by step instructions on how to get parsing for cisco ASA firewall logs

this is a production issue, so any help would be greatly appreciated

**Comment by Sahale** [comment]
Hello!
Filebeat already listens 514 port for "default" syslog. If you want to get and parse ASA logs, you should:

1. Set up another port (i.e. 2514);
2. Set up firewall in SO to allow send traffic from ASA to this port;
3. Reconfigure ASA to send traffic to new port;
4. See parsed traffic in Kibana :)
